roku_dev_cli/roku_dev_cli.py

```diff
@@ -108,7 +108,7 @@ def run(self):
         try:
             self.session = telnetlib.Telnet(self.ip, self.port)
             while True:
-                text = self.session.read_very_eager()
+                text = self.session.read_until(b"\n", 1)
```

Comment:
Is sending this as binary required?  If not, what's the difference?

Reply:
Unfortunately it is.  I tried using "\n" and got a runtime error saying I was supposed to provide a byte string, so I did 🤷‍♂️